applications/ws-diseases-ner/config.json

```
@@ -0,0 +1,21 @@
+{
+	"environnement": {
+		"EZS_TITLE": "diseases-ner",
+		"EZS_DESCRIPTION": "Reconnaissance d'entités nommées de maladies",
+		"EZS_METRICS": true,
+		"EZS_CONCURRENCY": 1,
+		"EZS_NSHARDS": 32,
+		"EZS_CACHE": true,
+		"NODE_OPTIONS": "--max_old_space_size=1024",
+		"NODE_ENV": "production",
+		"PIP_NO_DEPENDENCIES": 0,
+		"PIP_DISABLE_PIP_VERSION_CHECK": 1,
+		"PIP_NO_CACHE_DIR": 0,
+		"TRANSFORMERS_CACHE": "/app/.cache"
+	},
+	"packages": [
+		"@ezs/core@3.1.0",
```

```suggestion
		"@ezs/core@3.2.2",
```

À vérifier sur https://github.com/Inist-CNRS/ezs